host-bin/edit-chroot

```diff
@@ -385,8 +432,22 @@ for NAME in "$@"; do
         fi
         echo -n "  Restoring $src to $CHROOT..." 1>&2
         mkdir -p "$CHROOT"
+        if [ -f "$src.part01" ]; then
+            tarsrc="`mktemp -u --tmpdir=/tmp 'crouton-restore.XXX'`"
+            mkfifo -m 600 "$tarsrc"
+            # Pipe through dd to eat the EOFs
+            cat "$src" "$src.part"* | dd status=none >> "$tarsrc" &
+            catpid="$!"
+            addtrap "rm -f '$tarsrc'; kill '$catpid' 2>/dev/null"
+        else
+            tarsrc="$src"
+            catpid=''
+        fi
         spinner 1 tar --checkpoint=200 --checkpoint-action=exec=echo \
-            --one-file-system -xaf "$src" -C "$CHROOT" --strip-components=1
+            --one-file-system -xaf "$tarsrc" -C "$CHROOT" --strip-components=1
```

Comment:
Argh, yeah.  OK.  I now have it trying --test-label with the different available parameters to see which one works, then using that for the actual extraction.